image/mk_europe_edition64.sh

```diff
@@ -6,12 +6,12 @@
 # Run this script as root.
 # Run with argument "dev" to not clone the stratux repository from remote, but instead copy this current local checkout onto the image
 set -x
-BASE_IMAGE_URL="https://downloads.raspberrypi.com/raspios_lite_arm64/images/raspios_lite_arm64-2024-03-15/2024-03-15-raspios-bookworm-arm64-lite.img.xz"
-ZIPNAME="2024-03-15-raspios-bookworm-arm64-lite.img.xz"
+BASE_IMAGE_URL="https://downloads.raspberrypi.com/raspios_lite_arm64/images/raspios_lite_arm64-2024-07-04/2024-07-04-raspios-bookworm-arm64-lite.img.xz"
+ZIPNAME="2024-07-04-raspios-bookworm-arm64-lite.img.xz"
 IMGNAME="$(basename $ZIPNAME .xz)"
 TMPDIR="$HOME/stratux-tmp"
 # REMOTE_ORIGIN=$(git config --get remote.origin.url) # would be nicer, but doesn't work with ssh clone..
-REMOTE_ORIGIN="https://github.com/b3nn0/stratux.git"
+REMOTE_ORIGIN="https://github.com/JonWilder/stratux.git"
```

Comment:
This is probably unintentional, so can't merge it this way.

Reply:
This was so that I could do a production build and it would clone my repo with my code changes.